api/clients/v2/verification/conversion_utils.go

```diff
@@ -307,3 +300,23 @@ func BlobCommitmentsBindingToInternal(
 
 	return blobCommitment, nil
 }
+
+// QuorumNumbersUint32ToUint8 accepts an array of uint32 quorum numbers, and converts it into an array of uint8 quorum
+// numbers.
+//
+// Returns an error if any quorum number is too large to fit into a uint8
+func QuorumNumbersUint32ToUint8(inputQuorums []uint32) ([]uint8, error) {
```

Comment:
Outside the scope for this PR but @litt3 why are we doing this? Shouldn't the offchain code use uint8s everywhere? Why are we using uint32s?